conf/nginx/templates/nginx.conf.web.https.default.template

```diff
@@ -128,6 +128,18 @@ server
 
     location /services/files/
     {
+        set $public_link 0;
+        set $AUTH_COOKIE_MISSING 0;
+        if ($uri ~* "^/services/files/link/.*$")  {
```

Comment:
With the new public folder access feature we changed the link used to download a file maintaining the backward compatibility with the old link (`/services/files/link`). So you need to accept also the new one: `/services/files/public/link/download/`

Reply:
now both /service/files/link and /service/files/public are managed as URLs that do not require redirect